starters/nuxt/pages/cart.vue

```diff
@@ -5,39 +5,43 @@
       <ul class="cart__items">
         <li
           v-for="lineItem in cart.lineItems"
-          :key="lineItem.id"
+          :key="lineItem.variantId"
           class="cart__item"
         >
+          <pre>{{ JSON.stringify(lineItem, null, 2) }}</pre>
```

Comment:
This needs to be removed

```suggestion
```

<img width="800" alt="cart page pre with line item json" src="https://user-images.githubusercontent.com/5732000/191772686-6fbc7fd4-ab55-4b28-bef1-095596a358a8.png">